test-common/src/main/java/com/gentics/mesh/test/docker/ElasticsearchContainer.java

```diff
@@ -89,11 +89,8 @@ public ExecResult execRootInContainer(String... command) throws UnsupportedOpera
 
 		dockerClient.execStartCmd(execCreateCmdResponse.getId()).exec(callback).awaitCompletion();
 
-		final ExecResult result = new ExecResult(stdoutConsumer.toString(outputCharset), stderrConsumer.toString(outputCharset));
```

Comment:
Why was this changed?

Reply:
I updated the "test-common" -> "org.testcontainers:testcontainers" dependency locally as older versions didn't work.
In the newer versions the `ExecResult` isn't available anymore.
It's currently also never used anywhere, which makes it safe to update imo.

Reply:
I confirm this - newer versions of `testcontainers` have modified API.